src/System.Private.ServiceModel/tools/PRService/pr.ashx

```diff
@@ -99,10 +99,15 @@ public class PullRequestHandler : IHttpHandler
             {
                 case "master":
                 case "origin/master":
+                case "origin/infrastructure":
                     success = SyncToBranch(branchString, result);
                     break;
+                case "origin/release/1.0.0-rc2":
+                    context.Response.Write(string.Format("Branch name specified, '{0}' will not be synced by this service<br/>", HttpUtility.HtmlEncode(branchString)));
+                    success = true; 
```

Comment:
I'm viewing the HTTP/200 as "the PR service did what it needed to do for a PR to start". In this case, it's a no-op, since the PR service, if synced to the rc2 branch, would essentially nuke itself. 

Actually upon further thinking, we probably don't need this case, because the sync-pr scripts couldn't be called anyway (they don't exist either).